webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -163,16 +156,28 @@ export default {
   },
   methods: {
     submit() {
+      let image = null
+      const { title, content, categoryIds } = this.formData
+      if (this.formData.image) {
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-18T16:10:39Z" title="Tuesday, February 18th 2020, 5:10:39 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -212,11 +207,10 @@ export default {
         title,
         content,
         image,
-        teaserImage,
-        imageAspectRatio,
         categoryIds,
         blurImage,
       } = this.form
+      if (image) image.blurred = blurImage
```


not sure, but `this.blurImage`?